plugins/jobs/web_client/stylesheets/jobListWidget.styl

```diff
@@ -49,3 +34,46 @@
 
 #g-job-checkbox-menu
   display inline
+
+a.g-job-checkbox-menu-link
+  display inline-block
+  overflow hidden
+  text-overflow ellipsis
+  white-space nowrap
+  color: #333
+
+.dropdown
```

Comment:
This should be a more specific selector.  As is, it will modify properties of any dropdown in the page.